Items/04_06_MachineGuns_HeavyWeps_&_RocketLaunchers.txt

```diff
@@ -94,6 +94,32 @@ Movement Speed Penalty: -2m
 * 4 out to 45m
 * 5 out to 80m
 
+==Frontier Flame Thrower FLMR-B $3880 ==
+
+Damage: 80 Special
+Mag Size:  8 units of 6 liters, Naphitic Gel. $320 per magazine

```

Comment:
I like the relatively low ammo capacity. This thing hits like a truck. Slowing the user down to reload often I think creates a beautiful cadence for combat.